react/features/gifs/components/web/GifsMenuButton.tsx

```diff
@@ -4,10 +4,14 @@ import { useDispatch, useSelector } from 'react-redux';
 
 import ReactionButton from '../../../reactions/components/web/ReactionButton';
 import { showOverflowDrawer } from '../../../toolbox/functions.web';
-import { setGifDrawerVisibility, setGifMenuVisibility } from '../../actions';
+import { setGifDrawerVisibility, setGifMenuVisibility, setGifOverflowMenuVisibility } from '../../actions';
 import { isGifsMenuOpen } from '../../functions.web';
 
-const GifsMenuButton = () => {
+interface IProps {
+    overflowMenu?: boolean;
```

Comment:
Can't we use useState instead? Feels weird to get the overflow menu and overflow drawer options differently.

Reply:
Good catch!

I believe what happened here is during the time I was resolving some conflicts something changed and I haven't resolved them correctly because we are still passing the overflow drawer prop directly to the component but in the same time we don't use it and use the selector.